Sources/MediaViewController.swift

```diff
@@ -74,10 +74,16 @@ public class VLCMediaViewController: UICollectionViewController, UISearchResults
         setupSearchController()
         setupNavigationBar()
         _ = (MLMediaLibrary.sharedMediaLibrary() as! MLMediaLibrary).libraryDidAppear()
+        themeDidChange()
     }
 
     @objc func themeDidChange() {
         collectionView?.backgroundColor = PresentationTheme.current.colors.background
+        guard let backgroundView = collectionView?.backgroundView as? VLCEmptyLibraryView else {
```

Comment:
Since you're changing the emptyview it would be better to move this code into the VLCEmptyLibraryView. The MedialibraryViewcontroller should not know or modify the properties of the Emptyview. 
The reason why you do this. is because you might want to reuse the Emptyview somewhere else and then the controller who is using emptyview would have to change these properties as well -> duplication of code. If you do it in the emptyview itself this code is easier to find and second the class is easier to be reused. 

Observe the notification there and add a themeDidChange method in VLCEmptyLibraryView to update these properties.